includes/compat/class-wc-stripe-email-admin-failed-refund.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * An email sent to the admin when a refund fails.
+ *
+ * @since 9.6.0
+ */
+class WC_Stripe_Email_Admin_Failed_Refund extends WC_Stripe_Email_Failed_Refund {
+	/**
+	 * Constructor
+	 */
+	public function __construct() {
+		parent::__construct();
+
+		$this->id          = 'failed_refund_admin';
```

Comment:
Should we use a more specific string to avoid possible collisions with other gateways?
```suggestion
		$this->id          = 'wc_stripe_failed_refund_admin';
```